STYLE.md

```diff
@@ -185,29 +185,27 @@ $ docker run --name docsbuilder \
 quay.io/coreosinc/coreos-pages-builder scripts/deploy stage
 ```
 
-### Example: Documenting Code Blocks
+### Example: Documenting code blocks
 
-Code blocks are similar to long command lines, but with a fenced look. To format a code block in a document, insert triple backticks ` ``` ``` `before and after the code block. Use language identifier to enable language detection and syntax highlighting in the code block. Do not add a newline after the code block, Markdown automatically does for you.
+To format a code block in a document, insert triple backticks ` ``` ` ` ``` ` on a new line before and after the code block. A tag, such as `yaml`, `sh`, `json`, or `ini`, can be placed after the opening backticks to declare the language in the block. Check the escaping of the literal ` ``` ` chars with the preview.
```

Comment:
the part beginning "Check the escaping..." was a note to you. Delete that last sentence.